### DISCUSSION TITLE
How to help this project

### DISCUSSION THREAD

**Discussion by patricklodder** [rosetta-dogecoin/rosetta-dogecoin#7] [Q&A]
- Check the [issues](https://github.com/rosetta-dogecoin/rosetta-dogecoin/issues) that are tagged with "help wanted", investigate it a bit and if you think you can do it, let us know you're working on it.
- You'll want to [fork](https://github.com/rosetta-dogecoin/rosetta-dogecoin/fork) this repository to your private space and work on it there. Once you're done, update your own repository and then make a pull request (PR) with this repository. We'll review your PRs before merging them into this repository's main branch and provide feedback and/or request changes if needed.
- Remember that the less we change, the easier it will to be maintain things over the years. So please try to keep your changes concise and please don't try to refactor things too much. 
- If you spot anything that is suboptimal, just follow bullet point 2 above and throw in a PR. This goes for code, scripts, tests and documentation alike.

If you have any questions on how to do the above, like how to fork or do a PR, just ask us here, we'll help you get set up.

**Comment by zaridan** [comment]
This is really exciting to see! Do you foresee anything that non-devs who have some familiarity with Git, PRs etc might be able to pitch in and do? I.e., any kind of 'grunt work' that we can help with to save devs for doing the big stuff?

**Comment by kitcateleven** [comment]
Yeah - tutorials, any kind of media to help users, I can help with.  Also
Graphics creation etc.  I actually write PR video programming for
brands.  Whatever is needed to strengthen the community, I am happy to help.

On Thu, Mar 18, 2021 at 11:00 PM TJ Baker ***@***.***> wrote:

> This is really exciting to see! Do you foresee anything that non-devs who
> have some familiarity with Git, PRs etc might be able to pitch in and do?
> I.e., any kind of 'grunt work' that we can help with to save devs for doing
> the big stuff?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rosetta-dogecoin/rosetta-dogecoin/discussions/7#discussioncomment-501375>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AS7OED7WAEVW5EV2CQADJKDTEK46DANCNFSM4ZNIOJLA>
> .
>


-- 
Cat Mayville
President, Spot Digital Media
404-357-4503

**Comment by Iron-Legion-007** [comment]
Hi all,
Our Reddit community of over 1 million strong supporters of Dogecoin would like to know what we can do to help you all improve Dogecoin so that we can send it to the moon and beyond. Please let us know what we can do to help support you all.

**Comment by AtomicManProductions** [comment]
Hello! I am very interested in helping with this project, but I don't think I have anywhere near the coding knowledge to contribute much. However, I have been holding my Dogecoin for several years now and would be happy to donate some to the cause!
